src/formatDate.js

```diff
@@ -8,7 +8,42 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let dateArr = [];
+  let resultString = '';
+  let index = 0;
+
+  dateArr = date.split(fromFormat[3]);
+
+  for (let i = 0; i < toFormat.length - 1; i++) {
+    const fromIndex = fromFormat.indexOf(toFormat[i]);
+
+    index++;
+
+    if (toFormat[i] === 'YY' && fromFormat.includes('YYYY')) {
+      const number = dateArr[index - 1];
+
+      resultString += number.slice(-2);
+    } else if (toFormat[i] === 'YYYY' && fromFormat.includes('YY')) {
+      const year = Number(dateArr[index - 1]);
+
+      if (dateArr[index - 1] === '00') {
+        resultString += '2000.';
+        continue;
+      } else {
+        const fullYear = year < 30 ? `20${year}` : `19${year}`;
+
+        resultString += fullYear;
+      }
```

Comment:
Here you're converting from `YY` to `YYYY` format. You're correctly adding `20` or `19` before the year depending on its value. However, there are two issues here. First, you're adding a dot after `2000` on line 30, but you're not adding any separator after the year on line 35. This can lead to inconsistent results. Consider using the new format separator instead of a dot. Second, you're using `continue` statement on line 31. This can lead to skipping the addition of the separator after the year. Consider removing the `continue` statement and handling the addition of the separator after the `if...else` block.